bin/browser.js

```diff
@@ -73,10 +73,15 @@ const callChrome = async () => {
             request.options.clip = await element.boundingBox();
         }
 
-        output = await page[request.action](request.options);
+        if(request.action == 'evaluate') {
+            output = await page.evaluate(request.options.pageFunction);
+        } else {
```

Comment:
Can you refactor this, so we don't need an `else` statement?

First thought: let's introduce a `getOutput` function. You can use an early return in that function.

Reply:
Can you explain more about `getOutput`?

You mean `request.action == 'getOutput'`?

Reply:
I implemented it on d33189d31bd05bc2a26edc851ea20cbca7c40e0b
Is it the function you said?